test/remote/base_path_tests.js

```diff
@@ -4,12 +4,12 @@
 
 var test = require('../ptaptest')
 var TestServer = require('../test_server')
-var Client = require('../client')
+var Client = require('../client/index')
```

Comment:
Interesting looks like it was automatically introduced by Webstorm during refactor, I'll cleanup.